src/models/Application.py

```diff
@@ -169,6 +169,13 @@ def get_all_applications_for_user(user_id):
         return db.session.query(Application) \
             .filter((Application.assigned_to == user_id) & ((Application.score != 0) | ((Application.score == 0) & (Application.last_modified > cutoff))))
```

Comment:
this one too, though I know I added it a long time ago

Reply:
Done